views/components/quickactions.blade.php

```diff
@@ -0,0 +1,50 @@
+@if(boolval($userSettings['use_alternative_actions_on_stock_overview_page']))
```

Comment:
This completely drops the current quick consume/open buttons (and doesn't move them to the dropdown/more menu or something like that). The new buttons also don't have tooltips like the current ones do. Dropping existing functionality, also when it's just for an "alternative", can't be called a solution just like ever.